Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -694,6 +694,15 @@ void JitArm64::DoJit(u32 em_address, JitBlock* b, u32 nextPC)
     if (!SConfig::GetInstance().bEnableDebugging)
       js.downcountAmount += PatchEngine::GetSpeedhackCycles(js.compilerPC);
 
+    // Skip calling UpdateLastUsed for lmw/stmw - it usually hurts more than it helps
+    if (op.inst.OPCD == 46 || op.inst.OPCD == 47)
```

Comment:
Shouldn't this conditional be inverted? (wish we had an enum for the opcodes, had to check whether 46 and 47 were really lmw/stmw)

Reply:
Ah, yes :(

The tricky part of making something like an enum is that many instructions are identified by not just the `OPCD` field but also other fields, and it would feel weird to add something that doesn't work for all instructions.

Reply:
Yeah, that's true... Maybe calling it PrimaryOpcode (to match the interpreter's terminology) would make it clear enough  :man_shrugging: either way, that's out of the scope of this PR.